docs/dev/connectors/filesystem_sink.md

```diff
@@ -117,11 +117,12 @@ input.addSink(sink);
 </div>
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
-val input: DataStream[Tuple2[IntWritable, Text]] = ...
+import org.apache.flink.api.java.tuple.Tuple2
+val input: DataStream[Tuple2[A, B]] = ??? //we need to use java Tuple2 for the SequenceFileWriter
```

Comment:
Add a blank before `we`. Not sure about the comment. How about: Use java Tuple2 if we want to use SequenceFileWriter.
We can also wait suggestions from the other reviewers. 

Thank you very much for the fix and update. 

Best, 
Hequn